docs/timeseries/lombscargle.rst

```diff
@@ -4,31 +4,40 @@
 Lomb-Scargle Periodograms
 *************************
 
-The Lomb-Scargle Periodogram (after Lomb [1]_, and Scargle [2]_)
-is a commonly-used statistical tool designed to detect periodic signals
-in unevenly-spaced observations.
-The :class:`~astropy.timeseries.LombScargle` class is a unified interface to several
-implementations of the Lomb-Scargle periodogram, including a fast *O[NlogN]*
-implementation following the algorithm presented by Press & Rybicki [3]_.
-
-The code here is adapted from the `astroml`_ package ([4]_, [5]_) and the `gatspy`_ package ([6]_, [7]_).
-For a detailed practical discussion of the lomb-scargle periodogram, with code
-examples based on astropy, see *Understanding the Lomb-Scargle Periodogram* [11]_, with associated code at https://github.com/jakevdp/PracticalLombScargle/.
+The Lomb-Scargle Periodogram (after Lomb [1]_, and Scargle [2]_) is a commonly
```

Comment:
periodogram wasn't capitalized mid sentence in the BLS docs above and I think it shouldn't be in this page either